docs/sources/developers/http_api/data_source.md

```diff
@@ -907,15 +907,16 @@ Content-Type: application/json
 {
    "queries":[
       {
+         "expr": "kube_pod_info{container="example"}",
```

Comment:
Hi @Keetmalin,

this example is a query for the test datasource and the CSV Metric Values scenario, like this: https://play.grafana.org/goto/hnIh4jFSR?orgId=1

Thus, the `expr` property is not needed here.

```suggestion
```